src/main/java/com/stripe/model/CustomerCollection.java

```diff
@@ -2,4 +2,28 @@
 
 package com.stripe.model;
 
-public class CustomerCollection extends StripeCollection<Customer> {}
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.net.ApiResource;
+import com.stripe.net.RequestOptions;
+import java.util.Map;
+
+public class CustomerCollection extends StripeCollection<Customer> {
+  /** Retrieves a specific transaction applied to a customer’s balance. */
+  public Customer retrieve(String id) throws StripeException {
+    return retrieve(id, (Map<String, Object>) null, (RequestOptions) null);
+  }
+
+  /** Retrieves a specific transaction applied to a customer’s balance. */
+  public Customer retrieve(String id, RequestOptions options) throws StripeException {
+    return retrieve(id, (Map<String, Object>) null, options);
+  }
+
+  /** Retrieves a specific transaction applied to a customer’s balance. */
+  public Customer retrieve(String id, Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url =
+        String.format("%s%s", Stripe.getApiBase(), String.format("%s/%s", this.getUrl(), id));
+    return ApiResource.request(ApiResource.RequestMethod.GET, url, params, Customer.class, options);
+  }
```

Comment:
@remi-stripe 
Actually just to flag this. We had discussion before (during revision schedule) if we should add a `list` method here. I now realized that the pagination is now through 
```
ResourceCollection collectionInstance = myGetCollection();    
Iterator<Resource> resourceIterator = collectionInstance.autoPagingIterable().iterator();
while (resourceIterator.hasNext()) {
    doSomething(resourceIterator.next());
}
```
So `list` on collection instance wouldn't be helpful? Should we still include this?

Reply:
Hmmm I'll be honest I don't really know. As usual, deferring to @ob-stripe